Dockerfile

```
@@ -0,0 +1,51 @@
+
+# Build the binary
+RUN make build
+
+# Runtime stage
+FROM alpine:3.19
+
+# Install runtime dependencies
+RUN apk add --no-cache ca-certificates=20241121-r1 tzdata=2025b-r0
+
+# Create non-root user
+RUN adduser -D -u 1000 cloudpull
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)